addons/account/models/chart_template.py

```diff
@@ -578,6 +582,131 @@ def _setup_utility_bank_accounts(self, template_code, company, template_data):
+        # - Check if there is any tax for this country and stop the process if yes
+        # - Retrieve the tax group and tax template data
+        # - Create accounts at most probable location in the CoA and assign those to fields requiring an account
+        # - Creates tax group and taxes with their ir.model.data
+
+        taxes_in_country = self.env['account.tax'].search([
+            ('country_id', '=', country.id),
+            ('company_id', '=', company.id)
+        ])
+        if taxes_in_country:
+            return
+
+        def create_foreign_tax_account(existing_account, additional_label):
+            new_code = self.env['account.account']._search_new_account_code(
+                existing_account.company_id,
+                len(existing_account.code),
+                existing_account.code[:-2]
+            )
+            return self.env['account.account'].create({
+                'name': f"{existing_account.name} - {additional_label}",
+                'code': new_code,
+                'account_type': existing_account.account_type,
+                'company_id': existing_account.company_id.id,
+            })
+
+        existing_accounts = {}  # caching the found/created accounts to avoid further DB calls
+        default_company_taxes = company.account_sale_tax_id + company.account_purchase_tax_id
+
+        def corresponding_tax_account(type_tax_use, foreign_tax_rep_line):
+            template_id = foreign_tax_rep_line['account_id']
+            if existing_account := existing_accounts.get(template_id):
+                return existing_account
+
+            sign_comparator = '<' if int(foreign_tax_rep_line.get('factor_percent', 100)) < 0 else '>'
+            minimal_domain = [
+                ('company_id', '=', company.id),
+                ('account_id', '!=', False),
+                ('factor_percent', sign_comparator, 0),
+            ]
+            additional_domain = [
+                ('tax_id.type_tax_use', '=', type_tax_use),
+                ('tax_id.country_id', '=', company.account_fiscal_country_id.id),
+                ('tax_id', 'in', default_company_taxes.ids),
+            ]
+
+            # Trying to find an account being less restrictive on each iteration until the minimum acceptable is reached
+            # If nothing is found, don't fill it to avoid setting a wrong account
+            similar_repartition_line = None
+            while not similar_repartition_line and additional_domain:
+                search_domain = minimal_domain + additional_domain
+                similar_repartition_line = self.env['account.tax.repartition.line'].search(search_domain, limit=1)
+                additional_domain.pop()
+
+            similar_account_id = None
+            if similar_repartition_line:
+                local_tax_account = similar_repartition_line.account_id
+                similar_account_id = create_foreign_tax_account(local_tax_account, _("Foreign tax account (%s)", country.code)).id
```

Comment:
Yes, I talked about it with OCO.
We could just create one as we do for tax group and cash-basis accounts on taxes but setting stuff that is likely to be wrong might be worst than not setting anything at all.
Not setting it we "force" the user to acknowledge that he has something to do. (hence the comment at the beginning of the main function)
Personal note: the configuration will be working and the user might not even notice it's wrong.
I think TSB would say, as OCO, that we aren't responsible for the configuration by itself. We crunch a big bunch of work, they're supposed to do the last extra-mile.

But anyway, if there is any CoA installed on the company, this will never happen.
The only case I can think of is if the company not having any CoA on it (ie we don't have any localization for Malta and [people had trouble installing `l10n_eu_oss`](https://github.com/odoo/odoo/pull/120077/commits/5878fa117afb5c084e2f908a1ef6d89b2003c896)) nor any tax setup by the user.